packages/theming/apple-theme/src/appleTheme.colors.macos.ts

```diff
@@ -0,0 +1,223 @@
+    ghostDisabledBackground: 'transparent',
+    ghostDisabledBorder: 'transparent',
+    ghostDisabledContent: fluentApple.communicationBlueTint20,
+    ghostDisabledIcon: fluentApple.communicationBlueTint20,
+
+    brandBackground: fluentApple.communicationBlue,
+    brandBorder: 'transparent',
+    brandContent: appleSemantic.controlTextColor,
+    brandIcon: appleSemantic.controlTextColor,
+
+    brandHoveredBackground: fluentApple.communicationBlue,
+    brandHoveredBorder: 'transparent',
+    brandHoveredContent: appleSemantic.controlTextColor,
+    brandHoveredIcon: appleSemantic.controlTextColor,
+
+    brandFocusedBackground: fluentApple.communicationBlue,
+    brandFocusedBorder: 'transparent',
+    brandFocusedContent: appleSemantic.controlTextColor,
+    brandFocusedIcon: appleSemantic.controlTextColor,
+
+    brandPressedBackground: fluentApple.communicationBlueTint10,
+    brandPressedBorder: 'transparent',
+    brandPressedContent: appleSemantic.controlTextColor,
+    brandPressedIcon: appleSemantic.controlTextColor,
+
+    // TODO System Effect
+    brandDisabledBackground: fluentApple.gray100,
+    brandDisabledBorder: 'transparent',
+    brandDisabledContent: appleSemantic.controlTextColor,
+    brandDisabledIcon: appleSemantic.controlTextColor,
+
+    // TODO What is checked?
+    buttonCheckedBackground: appleSemantic.selectedContentBackgroundColor,
+    buttonCheckedContent: appleSemantic.controlTextColor,
+    buttonCheckedHoveredBackground: appleSemantic.selectedContentBackgroundColor,
+    buttonCheckedHoveredContent: appleSemantic.controlTextColor,
+
+    brandCheckedBackground: fluentApple.communicationBlueTint10,
+    brandCheckedContent: appleSemantic.controlTextColor,
+    brandCheckedHoveredBackground: fluentApple.communicationBlueTint10,
+    brandCheckedHoveredContent: appleSemantic.controlTextColor,
+
+    ghostCheckedBackground: 'transparent',
+    ghostCheckedContent: fluentApple.communicationBlueTint20,
+    ghostCheckedHoveredBackground: 'transparent',
+    ghostCheckedHoveredContent: fluentApple.communicationBlueTint20,
+    ghostCheckedHoveredBorder: 'transparent',
+
+    // TODO What to do with secondary?
+    ghostSecondaryContent: appleSemantic.secondaryLabelColor,
+    ghostFocusedSecondaryContent: appleSemantic.secondaryLabelColor,
+    ghostHoveredSecondaryContent: appleSemantic.secondaryLabelColor,
+    ghostPressedSecondaryContent: appleSemantic.secondaryLabelColor,
+
+    brandSecondaryContent: appleSemantic.secondaryLabelColor,
+    brandFocusedSecondaryContent: appleSemantic.secondaryLabelColor,
+    brandHoveredSecondaryContent: appleSemantic.secondaryLabelColor,
+    brandPressedSecondaryContent: appleSemantic.secondaryLabelColor,
+
+    buttonDisabledSecondaryContent: appleSemantic.tertiaryLabelColor,
+    buttonHoveredSecondaryContent: appleSemantic.tertiaryLabelColor,
+    buttonPressedSecondaryContent: appleSemantic.tertiaryLabelColor,
```

Comment:
These mappings are somewhat iffy, since we don't really have an equivalent of "secondary" for any of the FluentUI Apple controls

Reply:
for the tokens that doesn't make sense to the platform, I am curious to hear how to no-op it or flag it somehow like "this token doesn't make sense for the platform. don't use"

Reply:
Even if none of our FluentUI Apple controls have "secondary" content on them, there are JS components that do, so we would need to set something on them.

To not use specific tokens on a platform would need us to either:
- Don't set the tokens at all. This means they inherit from whatever the parent theme is, which could lead to unintended styling, and seems bad.
- Adding platform specific implementations of those JS controls that don't use the offending tokens. This seems like a bad idea because differing the API of a compoment that much may get confusing
- Mapping the offending tokens to some other token (like buttonBackgroundHover maps to buttonBackgroundRest). This is what I settled on to ensure that we control the styling to do what we want.